themes/bootstrap3/templates/RecordDriver/EDS/core.phtml

```diff
@@ -77,10 +77,33 @@
       <caption class="sr-only"><?=$this->transEsc('Bibliographic Details')?></caption>
       <?php foreach ($items as $key => $item): ?>
         <?php if (!empty($item['Data'])): ?>
-        <tr>
-          <th><?=empty($item['Label']) ? '' : $this->transEsc($item['Label']) . ':'?></th>
-          <td class="record__biblio-value"><?=$this->driver->linkUrls($item['Data'])?></td>
-        </tr>
+          <?php if ('Au' == $item['Group'] && strtolower($edsConfig->AuthorDisplay->ResultListFormat ?? 'Long') == 'short'): ?>
```

Comment:
I don't think $edsConfig is defined in the context of this template, unless I am overlooking something.

Also, the .ini file says the default is Short, but the default here is Long. We should pick one and be consistent.

Reply:
thanks @demiankatz ! sorry about the delay.

I've pushed a follow-up addressing both of your points here.